docs/source/AdministratorGuide/Tutorials/dmsWithTs.rst

```diff
@@ -12,31 +12,427 @@ You should:
 * have installed the DFC using the tutorial (:ref:`tuto_install_dfc`).
 * have followed the tutorial on identity management (:ref:`tuto_managing_identities`)
 * have installed the RMS using the tutorial (:ref:`tuto_install_rms`)
+* have installed the TS using the tutorial (:ref:`tuto_install_ts`)
 
 
-Tutorial goal
+Tutorial Goal
 =============
 
-The aim of the tutorial is to demonstrate how large scale data management operations (removals, replications, etc) can be achieved using the Transformation System.
-By the end of the tutorial, you will be able to:
+The aim of the tutorial is to demonstrate how large scale data management operations (removals, replications, etc.) can
+be achieved using the Transformation System.  By the end of the tutorial, you will be able to:
 
 * Submit simple transformation for manipulating a given list of files
 * Have transformations automatically fed thanks to metadata
-* Write your own plugin for TransformationSystem
+* Write your own plugin for the TransformationSystem
 
+The transformations can be monitored and controlled with the ``Transformation Monitor`` in the ``WebApp`` when you use
+the ``dirac_prod`` group.
 
-More links
+
+More Links
 ==========
 
 * :ref:`adminTS`
 
 
-Installing the RequestManagementSystem
-======================================
+Creating a Replication Transformation
+=====================================
+
+.. highlight:: console
+
+This section is to be performed as ``diracuser`` with a proxy in ``dirac_prod`` group.
+
+First we need to create some files and upload them to ``StorageElementOne``::
+
+  [diracuser@dirac-tuto ~]$ for ID in {1..10}; do echo "MyContent $ID" > File_${ID} ; dirac-dms-add-file /tutoVO/data/Trans_01/File_${ID} File_${ID} StorageElementOne ; done
+
+Then we create the list of LFNs we just uploaded::
+
+  [diracuser@dirac-tuto ~]$ dirac-dms-user-lfns -b /tutoVO/data/Trans_01
+  Will search for files in /tutoVO/data/Trans_01
+  /tutoVO/data/Trans_01: 10 files, 0 sub-directories
+  10 matched files have been put in tutoVO-data-Trans_01.lfns
+
+The easiest way to create a transformation to replicate files is by using the ``dirac-transformation-replication``
+command::
+
+  [diracuser@dirac-tuto ~]$ dirac-transformation-replication 0 StorageElementTwo --SourceSEs StorageElementOne -x
```

Comment:
Add a link to the command doc

Reply:
commands do not yet have the label added to the rst files, and some commands do not have entries, only in integration :(

Reply:
:'( We should get v7r0 out quickly...